paper/paper.md

```diff
@@ -0,0 +1,203 @@
+
+**PyElli** supports multiple solving algorithms with different characteristics.
+Currently, two algorithms using different formulations are available: a fast algorithm based on a 2x2 matrix formulation [@byrnes2020multilayer] and a more complex 4x4 matrix formulation [@Berreman72; @berreman4x4_doku; @berreman4x4_software].
+The 2x2 matrix algorithm divides the light into two perpendicular linearly polarized beams, which are solved separately.
+One of its limitations is eliminating the possibility to include birefringent materials.
+The 4x4 matrix approach fully solves Maxwell's equations.
+These equations describe the complete electromagnetic field inside each layer of the sample and couple these together using the matrix formalism.
+This allows finding solutions to more complex problems such as anisotropic materials, active media or magneto-optic samples.
+
+**PyElli** ensures fast processing through fully vectorized algorithms for multiple wavelengths and by leveraging numerical algebra libraries like [NumPy](https://numpy.org) [@harris2020array] and [SciPy](https://scipy.org) [@2020SciPy-NMeth].
+Together, these runtime advantages enable the practical use of advanced fitting algorithms such as global optimizers while maintaining reasonable evaluation times.
+As a result, they enable integrated in-situ monitoring and real-time data analysis of overlayer growth.
+Furthermore, the use of Python and vectorization libraries also facilitates the development of artificial intelligence-based SE data analysis.
+
+# Statement of need
+
+The importance of publishing data according to the FAIR principles is growing [@Wilkinson2016].
+Many research journals already require authors to add supporting data, and there is a growing expectation from sponsors for institutes and researchers to implement data governance.
+The FAIR principles have recently been extended to apply to research software as well since reproducing data requires not only the data itself but also the software used to create it [@Barker2022].
+Producing FAIR data and using a FAIR and open analysis pipeline is especially important for SE, as the results are tightly related and dependent on the algorithms and models used for evaluation.
+
+An open-source toolkit has many inherent benefits over proprietary software.
+For SE, optical models vary between manufacturers and translation can be difficult without comprehensive documentation.
+**PyElli's** open-source nature makes optical models extendable, auditable, and fully comprehensive.
+Each version of **PyElli** is associated with a DOI and a Zenodo upload, allowing for reliable referencing and reproducibility of analysis results.
+It supports [NXellipsometry](https://fairmat-experimental.github.io/nexus-fairmat-proposal/9636feecb79bb32b828b1a9804269573256d7696/classes/contributed_definitions/NXellipsometry.html#nxellipsometry), a recent advancement in SE data standardization.
```

Comment:
The link is outdated. This would be the official one: https://manual.nexusformat.org/classes/contributed_definitions/NXellipsometry.html and this the latest dev https://fairmat-nfdi.github.io/nexus_definitions/classes/contributed_definitions/NXellipsometry.html#nxellipsometry .

However, I'm not sure we still support the latest version so we might have to leave this part out or at least check that it still works (I'm not sure if there were relevant changes).